app/src/main/java/com/spielpark/steve/bernieapp/fragments/NavigationDrawerFragment.java

```diff
@@ -241,17 +241,6 @@ public void onConfigurationChanged(Configuration newConfig) {
     }
 
     @Override
-    public void onCreateOptionsMenu(Menu menu, MenuInflater inflater) {
-        // If the drawer is open, show the global app actions in the action bar. See also
-        // showGlobalContextActionBar, which controls the top-left area of the action bar.
-        if (mDrawerLayout != null && isDrawerOpen()) {
-            inflater.inflate(R.menu.global, menu);
-            showGlobalContextActionBar();
-        }
```

Comment:
Nope, should work fine. Only thing is the app name won't change when the drawer is opened.